scss/pixel-arts/kirby.scss

```
@@ -0,0 +1,38 @@
+.kirby {
```

Could you confirm it again? 🤔 
```suggestion
.nes-kirby {
```

---

For some reason my login don't went in so the push failed, just checked and now it should be good.